frontend/packages/console-shared/src/components/dashboard/status-card/AlertItem.tsx

```diff
@@ -42,6 +44,10 @@ export const StatusItem: React.FC<StatusItemProps> = ({ Icon, timestamp, message
 };
 
 const AlertItem: React.FC<AlertItemProps> = ({ alert }) => {
+  const actionsExtensions = useExtensions(isAlertAction);
+  actionsExtensions.forEach(({ properties }) =>
+    alertActions.set(properties.alert, properties.value),
+  );
   const action = alertActions.get(alert.rule.name);
```

Comment:
would be better if we change `alertActions` to function. Something like
```javascript
const getAlertActions = (acionsExtensions: AlertAction[]) => ....
```

and use it everywhere like
```javascript
const actionsExtensions = useExtensions(isAlertAction);
const action = getAlertActions(actionsExtensions).get(alert.rule.name);
```

that way a component does not always have to do `forEach` on its own